mailbox/store/src/main/java/org/apache/james/mailbox/store/json/event/dto/MailboxSessionIdDataTransferObject.java

```diff
@@ -0,0 +1,39 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.mailbox.store.json.event.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class MailboxSessionIdDataTransferObject {
```

Comment:
doesn't make sense of an Id

Reply:
I don't understand, SessionId need to by carried in Event DTO, but we dont need whole mailbox session, then I removed MailboxSessionDTO and created new one for only ID, what doesnt make sense?

Reply:
Ok, I was wrong about that. It will disappear with the new serialization module soon anyway.